traits/util/trait_documenter.py

```diff
@@ -78,7 +79,7 @@ def import_object(self):
 
         """
         try:
-            __import__(self.modname)
+            import_module(self.modname)
             current = self.module = sys.modules[self.modname]
             for part in self.objpath[:-1]:
```

Comment:
I think this needs fixing too.

Reply:
Agreed, though I'm happy for that to happen in a separate PR.

Reply:
Turns out it's OK because it was calling `__import__` and then looking in `sys.modules` rather than walking the tree.  The `objpath` stuff is code which depends on sphinx autodoc; I have made a slight improvement to this as we can replace the `sys.modules` lookup with just the `import_module`.